### ISSUE TITLE
Grav not working due to RocketTheme

### ISSUE THREAD

**Issue by ghtmtt** [getgrav/grav#2850]
I'm deploying grav website on my local computer before to pushing to the server. I'm getting this error when using `php -S localhost:8080 system/router.php`:
```

Server Error

Sorry, something went terribly wrong!
E_WARNING - require_once(): \RocketTheme\Toolbox\StreamWrapper\ReadOnlyStream::stream_set_option is not implemented!
```

**Comment by rhukster** [comment]
It seems like this is PHP 7.4 related.  @mahagr can you take a look?

**Comment by mahagr** [comment]
@rhukster This has already been fixed in the last released version of the library. Grav 1.6 is currently locked to use older version, though.

**Comment by mahagr** [comment]
See also https://github.com/getgrav/grav/issues/2832

**Comment by ghtmtt** [comment]
ok so the newer version of grav (when released) contains the fix?

**Comment by rhukster** [comment]
Fixed for next release.

**Comment by ghtmtt** [comment]
@rhukster super thanks! 

Can i early test it with [https://github.com/getgrav/grav/releases/tag/1.7.0-rc.7](https://github.com/getgrav/grav/releases/tag/1.7.0-rc.7)?

**Comment by mahagr** [comment]
This has already been fixed ages ago in 1.7-rc, but now we also backported the change to 1.6.

**Comment by ghtmtt** [comment]
> This has already been fixed ages ago in 1.7-rc, but now we also backported the change to 1.6.

sounds great! So when the issue is closed the fix will be backported. Waiting for that. Thanks!
